### DISCUSSION TITLE
Are there any search events emitted so we can track the number of results?

### DISCUSSION THREAD

**Discussion by ianthetechie** [squidfunk/mkdocs-material#7374] [Q&A] [answered]
Context: I'm integrating Matamo analytics and want to track searches. I'm successfully doing it using the advice here: https://github.com/squidfunk/mkdocs-material/discussions/5732.

One of the Matomo analytics features is to track the number of results. Is there a way that I can do that besides resorting to DOM inspection (which seems super brittle)?

**Comment by squidfunk** [comment, marked as answer]
Currently not, but implementing this will be trivial once #6307 is out.

**Reply by ianthetechie** [reply]
Ok; will watch that task then. Thanks!
